wiki/People/Project_Loved_Team/en.md

```diff
@@ -0,0 +1,47 @@
+# Project Loved Team
+
+The **Project Loved Team** are members of the osu! community who manage [Project Loved](/wiki/Project_Loved), the voting platform that promotes beatmaps to the [Loved](/wiki/Beatmaps#loved) category.
```

Comment:
```suggestion
The **Project Loved Team** are members of the osu! community who manage [Project Loved](/wiki/Project_Loved), the voting project that promotes beatmaps to the [Loved](/wiki/Beatmaps#loved) category.
```
dont know if necessary

Reply:
think "platform" makes more sense (the "project" part has a wider scope than voting but the voting is all that's important for public)